app/build.gradle

```diff
@@ -172,6 +172,53 @@ dependencies {
     implementation Deps.androidx_constraintlayout
     implementation Deps.androidx_preference
     implementation Deps.google_material
+
+
+    implementation 'com.squareup.okhttp3:okhttp:3.11.0'
+    implementation 'com.squareup.picasso:picasso:2.71828', {
+        // We temporarily don't include the packaged the support libs to avoid version mismatch.
+        // This can cause runtime errors so we should remove this ASAP! #883?
+        exclude group: 'com.android.support', module: 'exifinterface'
+        exclude group: 'com.android.support', module: 'support-annotations'
+        exclude group: 'com.android.support', module: 'support-compat'
+        exclude group: 'com.android.support', module: 'support-fragment'
+    }
+
+    androidTestImplementation 'com.android.support.test.uiautomator:uiautomator-v18:2.1.3'
```

Comment:
Let's manage version numbers in a central place similar to `espresso_version` or better https://github.com/mozilla-mobile/reference-browser/blob/master/buildSrc/src/main/java/Dependencies.kt#L6